design/cryptography.md

```diff
@@ -54,6 +55,23 @@ An additional complexity is that Obscuro has the requirement to support temporar
   - Transactions without Revelation Period specified are encrypted by default using the smallest unit of time revelation period
```

Comment:
this is not correct. 
Transactions don't have a revelation period as an intrinsic property of them.
It's only the smart contracts that have this property

Reply:
That was not from the whitepaper, it was my proposal. Where can I read about the expected revelation design ? I didn't see anything in regard to this on the white paper, but I might have missed it.